src/main/java/me/zeroeightsix/kami/module/modules/chat/AutoExcuse.kt

```diff
@@ -28,6 +32,8 @@ object AutoExcuse : Module() {
             "I'm not using $CLIENT_NAME client"
     )
 
+    private val backup = excuses.toMutableList()
```

Comment:
Instead of storing 2 copies by default, why not have an empty mutable list as excuses and make backup have the original list?

This will save a fraction of ram at runtime but it doesn't hurt anybody